src/chains/filecoin/filecoin/src/blockchain.ts

```diff
@@ -611,13 +613,20 @@ export default class Blockchain extends Emittery.Typed<
             continue;
           }
 
-          // TODO: figure out nonce/error logic
```

Comment:
I went through the nonce logic when errors occur in lotus, and the current implementation is valid, so this TODO was removed